crates/compiler/test_syntax/tests/snapshots/pass/where_implements_lots_of_newlines.expr.formatted.roc

```diff
@@ -0,0 +1,7 @@
+T :
+    [
+    ] #
+        where e
+        implements
```

Comment:
I feel like in other places implements is indented from the where when forced to a newline

Reply:
Right now, the only constraint is that in a where, everything after the original type (`[]` in this case) is indented further.

Reply:
(Definitely open to tweaking this!)